include/irr/asset/bawformat/blobs/MeshBufferBlob.h

```diff
@@ -32,22 +37,18 @@ struct IRR_FORCE_EBO MeshBufferBlobV0 : TypedBlob<MeshBufferBlobV0, ICPUMeshBuff
 	uint32_t baseInstance;
 	uint32_t primitiveType;
 	uint32_t posAttrId;
+	bool isRightHandedCoordinateSystem;
 } PACK_STRUCT;
 #include "irr/irrunpack.h"
-static_assert(sizeof(MeshBufferBlobV0::mat)==197, "sizeof(MeshBufferBlobV0::mat) must be 197");
+static_assert(sizeof(MeshBufferBlobV3::mat)==197, "sizeof(MeshBufferBlobV0::mat) must be 197");
 static_assert(
-    sizeof(MeshBufferBlobV0) ==
-    sizeof(MeshBufferBlobV0::mat) + sizeof(MeshBufferBlobV0::box) + sizeof(MeshBufferBlobV0::descPtr) + sizeof(MeshBufferBlobV0::indexType) + sizeof(MeshBufferBlobV0::baseVertex)
-    + sizeof(MeshBufferBlobV0::indexCount) + sizeof(MeshBufferBlobV0::indexBufOffset) + sizeof(MeshBufferBlobV0::instanceCount) + sizeof(MeshBufferBlobV0::baseInstance)
-    + sizeof(MeshBufferBlobV0::primitiveType) + sizeof(MeshBufferBlobV0::posAttrId),
+    sizeof(MeshBufferBlobV3) ==
+    sizeof(MeshBufferBlobV3::mat) + sizeof(MeshBufferBlobV3::box) + sizeof(MeshBufferBlobV3::descPtr) + sizeof(MeshBufferBlobV3::indexType) + sizeof(MeshBufferBlobV3::baseVertex)
+    + sizeof(MeshBufferBlobV3::indexCount) + sizeof(MeshBufferBlobV3::indexBufOffset) + sizeof(MeshBufferBlobV3::instanceCount) + sizeof(MeshBufferBlobV3::baseInstance)
+    + sizeof(MeshBufferBlobV3::primitiveType) + sizeof(MeshBufferBlobV3::posAttrId) + sizeof(MeshBufferBlobV3::isRightHandedCoordinateSystem),
     "MeshBufferBlobV0: Size of blob is not sum of its contents!"
 );
 
-using MeshBufferBlobV1 = MeshBufferBlobV0;
-using MeshBufferBlobV2 = MeshBufferBlobV1;
```

Comment:
still need aliases for Ver2 to Ver0

Reply:
Matt is it neseccary if there is no change since Ver0 to Ver2 up? I have exchaned meshbuffers using 0 version (that are the same till V3) for version 2 and I find it valid, it compiles

Reply:
We always provide all the V0 to VLatest at least as aliases.